### PR TITLE
[contacts] Add note to generated entitlements file

### LINKED ISSUE #5949: [expo-contacts] getContactsAsync hangs on iOS 13
### Environment

```
  Expo CLI 3.0.9 environment info:
    System:
      OS: macOS 10.14.5
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 12.9.1 - /usr/local/bin/node
      Yarn: 1.17.3 - /usr/local/bin/yarn
      npm: 6.10.3 - /usr/local/bin/npm
    IDEs:
      Android Studio: 3.5 AI-191.8026.42.35.5791312
      Xcode: 11.1/11A1027 - /usr/bin/xcodebuild
    npmPackages:
      @storybook/react-native: ^5.1.9 => 5.1.10 
      expo: ^34.0.1 => 34.0.3 
      react: 16.8.3 => 16.8.3 
      react-native: 0.59.8 => 0.59.8 
      react-navigation: ^3.0.9 => 3.11.1 
    npmGlobalPackages:
      expo-cli: 3.0.9
```

We're running Standalone.

### Steps to Reproduce

1. Request permission to contacts

    `Permissions.getAsync(Permissions.CONTACTS)`

2. Grab non-image fields

    ```
    export const nonImageFields = Object.values(Contacts.Fields).filter(
      field =>
        field !== Contacts.Fields.Image && field !== Contacts.Fields.RawImage,
    )
    ```

3. Fetch contacts async
    `const { data: allContacts } = await Contacts.getContactsAsync({fields: nonImageFields,})`

### Expected Behavior

At some point, the async function will return, meaning a log statement the line below would fire and the allContacts list would be populated (assuming > 0 contacts).

### Actual Behavior

The async function hangs indefinitely. The log statement on the next line is never fired, moreover no line below is ever executed as it's still awaiting `getContactsAsync`.

### Notes

If I only fetch `"emails"` as a field it works fine. This makes me think that some fields no longer work in iOS 13.

### PR DESCRIPTION
**Pull request by lukmccall**
# Why

Resolve #5949.

# How

Now, users can set `com.apple.developer.contacts.notes` entitlement (It's connected to https://github.com/expo/expo/commit/882d80bf97b50d7cade84bf4a3f09d2af702bdfc).

# Test Plan

N/D

### CODE REVIEW

File: docs/pages/versions/unversioned/workflow/configuration.md [hunk 1]
```diff
@@ -431,6 +431,14 @@ Configuration for how and when the app should request OTA JavaScript updates
     */
     "usesAppleSignIn": BOOLEAN,
 
+    /*
+      A boolean value indicating if the app may access the notes stored in contacts.
+      See Contacts docs for details.
+
+      ExpoKit: use Xcode to set this.
+    */
+    "note": BOOLEAN,
```
Comment (tsapeta) [inline docs/pages/versions/unversioned/workflow/configuration.md]: name of this configuration should be self-descriptive, `note` tells nothing to the users

**Review by tsapeta — CHANGES_REQUESTED** [review]

**Review by sjchmiela — APPROVED** [review]
LGTM! Has the Universe PR been merged?

File: docs/pages/versions/unversioned/sdk/contacts.md [hunk 2]
```diff
@@ -719,7 +719,7 @@ const contactField = Contact.Fields.FirstName;
...
 | ImageAvailable          | `'imageAvailable'`           | ✅  | ✅      |      |
 | Image                   | `'image'`                    | ✅  | ✅      |      |
-| Note                    | `'note'`                     | ⚠️  | ✅      | On iOS 13 and above, your app must have the `com.apple.developer.contacts.notes` entitlement. You must contact Apple and receive approval for this entitlement. Read [the Apple docs](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_contacts_notes) to learn more. |
+| Note                    | `'note'`                     | ⚠️  | ✅      | On iOS 13 and above, your app must have the `com.apple.developer.contacts.notes` entitlement. You must contact Apple and receive approval for this entitlement. Read [the Apple docs](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_contacts_notes) to learn more. Expo client doesn't have this entitlement. However, you can add this to a standalone app. To do this you need to set the `note` key to true in your app.json file as specified [here](https://docs.expo.io/versions/latest/workflow/configuration/#ios). |
```
Comment (sjchmiela) [inline docs/pages/versions/unversioned/sdk/contacts.md]:
```suggestion
| Note                    | `'note'`                     | ⚠️  | ✅      | On iOS 13 and above, your app must have the `com.apple.developer.contacts.notes` entitlement. You must contact Apple and receive approval for this entitlement. Read [the Apple docs](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_contacts_notes) to learn more. Expo client doesn't have this entitlement. However, you can add this to a standalone app. To do this you need to set the `accessesContactNotes` key to true in your app.json file as specified [here](https://docs.expo.io/versions/latest/workflow/configuration/#ios). |
```

**Review by sjchmiela — APPROVED** [review]
Just one suggestion and then 👍

**Pull request merged**